### PR TITLE
Chore/fixes plan table

### PR DESCRIPTION
**Pull request by aatauil**
- fix typo nummer => nummers
- set default to 0 in convertInputValuesToNumbers function
- save totalContribution in database

To test totalContribution, create some forms with values in the table and after saving you can execute the following query to view the values in the database:

```
      PREFIX foaf: <http://xmlns.com/foaf/0.1/>
      PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
      PREFIX subsidie: <http://data.vlaanderen.be/ns/subsidie#>
      PREFIX lblodSubsidie: <http://lblod.data.gift/vocabularies/subsidie/>
      PREFIX transactie: <http://data.vlaanderen.be/ns/transactie#>
      PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
      PREFIX m8g: <http://data.europa.eu/m8g/>
      PREFIX dct: <http://purl.org/dc/terms/>
      PREFIX adms: <http://www.w3.org/ns/adms#>
      PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
      PREFIX schema: <http://schema.org/>
      PREFIX planSamenleven: <http://lblod.data.gift/vocabularies/subsidie/plan-samenleven/>

      SELECT DISTINCT ?form ?totalContribution
      WHERE {
        ?subsidie a subsidie:SubsidiemaatregelConsumptie ;
                  transactie:isInstantieVan <http://lblod.data.info/id/subsidy-measure-offers/a2d920b2-d266-4555-b023-63632997c406> ;
                  dct:source ?form .

        ?form lblodSubsidie:planLivingTogetherTable ?table.
        ?table planSamenleven:totalContribution ?totalContribution.
      }
```

### CODE REVIEW

File: addon/components/custom-subsidy-form-fields/plan-living-together-table/edit.hbs [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
       <li>In de vierde kolom ziet u het bedrag dat in aanmerking komt voor subsidiering door de Vlaamse overheid. Er wordt verwacht dat het bestuur voor deze acties evenveel financiert.</li>
-      <li>Geef in de vierde kolom aan de hand van cijfers aan wat de prioriteit is per actie waarop het bestuur bijkomend inzet. Wanneer er niet voldoende middelen zijn, zal de Vlaamse overheid aan de hand van deze prioritering bepalen naar welke acties de subsidies gaan. Wanneer jullie bestuur intekent voor bv. 5 acties, dan moeten de nummer 1 t.e.m. 5 verdeeld worden over deze acties.</li>
+      <li>Geef in de vierde kolom aan de hand van cijfers aan wat de prioriteit is per actie waarop het bestuur bijkomend inzet. Wanneer er niet voldoende middelen zijn, zal de Vlaamse overheid aan de hand van deze prioritering bepalen naar welke acties de subsidies gaan. Wanneer jullie bestuur intekent voor bv. 5 acties, dan moeten de nummers 1 t.e.m. 5 verdeeld worden over deze acties.</li>
```
Comment (Windvis) [inline addon/components/custom-subsidy-form-fields/plan-living-together-table/edit.hbs:8]: I think the show component also needs this fix?

**Review by Windvis — COMMENTED** [review]
Code looks good to me. I haven't tested it but I trust that it works if you tested it :smile:

File: addon/components/custom-subsidy-form-fields/plan-living-together-table/edit.hbs (hunk 1 above)
Comment (aatauil) [reply addon/components/custom-subsidy-form-fields/plan-living-together-table/edit.hbs:8]: indeed!

**Comment by Windvis** [comment]
the linter is also complaining due to Prettier so running `npm run lint:fix` should do the trick I think.

**Comment by aatauil** [comment]
Always forget that step, for some reason nothing pops up in my problems tab for this repo. the lint fix is now pushed :)

**Pull request merged**